README.md

```diff
@@ -249,6 +249,53 @@ the autocomplete/ctr file in your `.bashrc`, or manually like:
 $ source ./contrib/autocomplete/ctr
 ```
 
+### CRI
+
+`cri` is a [containerd](https://containerd.io/) plugin implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/cri-api/blob/master/pkg/apis/runtime/v1alpha2/api.proto).
+
+With it, you could run Kubernetes using containerd as the container runtime.
```

Comment:
I don't think this needs to be separated by a paragraph break from the opening sentence above; following re-wording suggested:

With the cri plugin you are able to use containerd as the container runtime for a Kubernetes cluster.